packages/data-explorer/src/views/MainView.vue

```diff
@@ -133,34 +143,31 @@ export default Vue.extend({
         this.deleteRow({ rowId: itemId })
       }
     },
-    async fetchViewData (tableName) {
-      if (this.tableName !== tableName) {
-        this.loading = true
-        await this.fetchTableMeta({ tableName })
-        if (this.isUserAuthenticated) {
-          this.fetchBreadcrumbs()
-        }
-        this.setTableName(tableName)
-      }
-      if (this.dataDisplayLayout === 'CardView') {
-        this.fetchCardViewData()
+    async paginateEntities () {
+      // The data-API uses zero-based page numbering, where pagination is one-based.
+      const pagination = { ...this.pagination, page: this.pagination.page - 1 }
+      const request = await this.fetchViewData({ tableName: this.$route.params.entity, pagination })
```

Comment:
could we move this into the store

Reply:
Replaced the fetchItems method with fetchViewData. No need for this method at all.